### PR TITLE
Adding Example 5

### LINKED ISSUE #145: New example problem for simulating concrete - soil direct shear type test (ex05)
The example focuses on applying normal pressure on a concrete block while it is subjected to displacement boundary conditions to simulate sliding behavior. Contact enforced through thin layer elements. Sensitivity studies performed to assess the shear stress-strain behavior in vicinity of foundation.

### PR DESCRIPTION
**Pull request by dhruvdattapurdue**
Direct shear test: Concrete block sliding on non-linear soil with thin layer contact formulation closes  #145

### CODE REVIEW

File: doc/content/examples/example5.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+of the interface once the yield point is reached. The pressure dependency of the backbone curve is turned on to ensure that the shear strength increases linearly with the normal pressure.
+
+!listing examples/ex05/3Dsliding_8psi.i
+         start=Materials
+         end=Preconditioning
+
+The function block includes the gravity (32.2 $ft/s^2$), depth and density (90.5 $lb/ft^3$) dependent functions for initial stress to create in-situ soil conditions. The properties of the user-defined backbone curve are modified in accordance with the normal pressure as shown below.
```
Comment (cbolisetti) [inline doc/content/examples/example5.md]: '... to create in-situ soil conditions in the soil domain.'

File: doc/content/examples/example5.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+The table below shows the backbone curve for F-55 Ottawa sand under varied loading conditions.
```
Comment (cbolisetti) [inline doc/content/examples/example5.md]: Please change this to a markdown table instead.

File: doc/content/examples/example5.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+       style=width:50%;margin-left:250px;float:center;
+       id=fig:Backbonecurve1
+       caption=Backbone curves of F-55 Ottawa sand under varied confining pressures.
+
+!media media/examples/F55OttawaSand_Mod_reduction.jpg
+       style=width:50%;margin-left:250px;float:center;
+       id=fig:Modreduction
+       caption=Modulus reduction curves of F-55 Ottawa sand.
+
+The averaged shear stress response at the interface is obtained at the interface by using the ElementAverageValue postprocessor. The results are in correspondence with theoretically computed values (see [fig:ex5_results1] [fig:ex5_results2] [fig:ex5_results3]). The Stress_zz pressure bulb distribution is theoretically computed using Newmarks and Boussinesq’s formulas for vertical stress profile beneath square footings, and the results match with the simulation values.
```
Comment (cbolisetti) [inline doc/content/examples/example5.md]: > The shear stress response at the interface is obtained at the interface using the ElementAverageValue postprocessor, which calculates the shear stress (averaged across the quadrature points) in each element. The results are similar to the theoretically computed values (see [fig:ex5_results1] [fig:ex5_results2] [fig:ex5_results3]). The Stress_zz pressure bulb distribution is theoretically computed using Newmarks and Boussinesq’s formulas for vertical stress profile beneath square footings, and the results match with the simulation values.

Comment (cbolisetti) [inline doc/content/examples/example5.md]: Also, what is the experimental simulation? Were there experiments on this?
